example/generator/odp_generator.c

```diff
@@ -458,7 +497,7 @@ static odp_packet_t setup_icmp_pkt_ref(odp_pool_t pool,
  */
 static int setup_icmp_pkt(odp_packet_t pkt,
 			  odp_pktout_config_opt_t *pktout_cfg,
-			  counters_t *counters)
+			  counters_t *counters, void *arg)
```

Comment:
I'd suggest to use `ODP_UNUSED` here, rather than just `(void) arg`.

Reply:
OK